packages/ffe-account-selector-react/src/subcomponents/input-field/InputField.js

```diff
@@ -101,6 +101,7 @@ class Input extends Component {
                         onMouseDown={this.onReset}
                         tabIndex={-1}
                         type="button"
+                        aria-label="reset"
```

Comment:
This text will be read out loud by screen readers and should follow the same language settings as the rest of the component. If you look in the [i18n](https://github.com/SpareBank1/designsystem/tree/develop/packages/ffe-account-selector-react/src/i18n) folder you'll find the languages and texts that are being used in the component. `RESET_SEARCH` looks promising to use here, and the label for the expand/collapse button could perhaps use `ACCOUNTSLIST_OPEN` and `ACCOUNTSLIST_CLOSE` depending on state? 

The [AccountNoMatch](https://github.com/SpareBank1/designsystem/blob/36dc3a255a300b3639b727ccdef8729c24a6a036/packages/ffe-account-selector-react/src/subcomponents/account/AccountNoMatch.js) component has an example on how to use the language file. You need to know the current `locale` though, which `Input` doesn't right now. From what I can see `Input` is only used from `BaseSelector`. `BaseSelector` gets the `locale` prop from `AccountSelector` and `AccountSelectorMulti`, it's just not used/listed in `propTypes`. So if you send the `locale` prop from `BaseSelector` to `Input` and use the language file this should be good to go :)